avocado/plugins/list.py

```diff
@@ -165,6 +168,10 @@ def configure(self, parser):
                             'Current: %(default)s')
         loader.add_loader_options(parser)
 
+        filtering = parser.add_argument_group('filtering parameters')
+        filtering.add_argument('--filter-by-tags', metavar='TAGS',
+                               action='append')
```

Comment:
yep, that's why I said I still prefer the `-`. So let's use it and please do mention in the documentation the `--filter-by-tags=-foo`. it is a bash issue, but it's super helpful when one forgets...